tests/data/pings.yaml

```diff
@@ -0,0 +1,9 @@
+customPing:
+    description:
+        This is a custom ping
+    include_client_id: false
+
+reallyCustomPing:
```

Comment:
Note that `reallyCustomPing` can't be used in `metrics.yaml` (for `send_in_pings`), because we require [snake case](https://github.com/mozilla/glean_parser/blob/30642e1d91ce7f88cca4beba76dd9f08f6f51383/glean_parser/schemas/metrics.1-0-0.schema.yaml#L175) for entries there. I think we'd need to be consistent between the two (i.e. enforce the same constraints).

Reply:
Good catch.  Definitely the intention is to use snake case here.  The unit test for catching this was wrong...  Should be fixed now.